components/screens/AppLoadingScreen.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react'
+import React, { Component } from 'react'
 import { ActivityIndicator, AsyncStorage, StatusBar, StyleSheet, View } from 'react-native'
 import { Button } from 'react-native-elements'
 
-export default class AppLoadingScreen extends React.Component {
+export default class AppLoadingScreen extends Component {
```

Comment:
I saw this in the official documentation https://facebook.github.io/react-native/.
It's seems logic to extends `Component` class because you imported it with `import React, { Component } from 'react'`.
And this `Component` class should be the only one to be `Component` it's like the mother of all the components, you can`t miss her